lib/views/create_livestream/active_stream.dart

```diff
@@ -1,9 +1,12 @@
 import 'dart:async';
+import 'dart:io' show Platform;
```

Comment:
`'dart:io'` and the `StateAsync` class both have a `sleep()` function, so I added `show Platform` to make sure that the `Platform` class is the only thing imported from that library.